### PR TITLE
Revised instructions on creating an environment

### PR DESCRIPTION
**Pull request by sungeunbae**
The steps described in this README is required to create a virtual environment on NeSI.
Found some incorrect stuffs while I walking through with Cesar and Andrew.

### CODE REVIEW

**Review by claudio525 — APPROVED** [review]

File: workflow/environments/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Requirements: Setup github SSH keys for maui,
 
 A user specific environment of workflow, qcore, IMCalc, Empirical Engine and Pre-processing 
 can be created as follows:  
-1) Go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy
+1) Log in to Maui, and go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy. If you don't have it, `git clone` now.
 2) Navigate to ".../slurm_gm_workflow/workflow/environments/org/nesi"
```
Comment (lispandfound) [inline workflow/environments/README.md:27]: This `...` is relative, but you don't say relative to where. In my case I:

1. Cloned `slurm_gm_workflow` in my home directory
2. Navigated to `slurm_gm_workflow/workflow/environments/org/nesi`

I think this should not have a relative path. Maybe explicitly state to work from `$HOME` and then navigate to 
`~/slurm_gm_workflow/workflow/environments/org/nesi`

File: workflow/environments/README.md [hunk 2]
```diff
@@ -20,13 +20,16 @@ Requirements: Setup github SSH keys for maui,
 
 A user specific environment of workflow, qcore, IMCalc, Empirical Engine and Pre-processing 
 can be created as follows:  
-1) Go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy
+1) Log in to Maui, and go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy. If you don't have it, 
+```bash
+git clone git@github.com:ucgmsim/slurm_gm_workflow.git
+```
 2) Navigate to ".../slurm_gm_workflow/workflow/environments/org/nesi"
 3) Load the correct version of Python module. (** IMPORTANT **). For example, Python 3.9, try the following.
```
Comment (lispandfound) [inline workflow/environments/README.md:28]: NB: You should use `**IMPORTANT**` not `** IMPORTANT **` to bold the text. 

**IMPORTANT** vs ** IMPORTANT **

Comment (lispandfound) [reply workflow/environments/README.md:28]: If bold is actually what you want, that is.

**Review by lispandfound — CHANGES_REQUESTED** [review]

**Review by joelridden — APPROVED** [review]
Good changes, what Jake said about some other sections also could be good to change as well. But approve the rest.

Also will need the Black upgrade run over this code, hasnt been changed in a while

**Comment by sungeunbae** [comment]
All Python changes are unrelated to the update to the README - just to keep Black happy.

**Review by joelridden — APPROVED** [review]

**Review by claudio525 — APPROVED** [review]

**Review by lispandfound — APPROVED** [review]

**Comment by sungeunbae** [comment]
@AndrewRidden-Harper  You have followed this README to set up the virtual enviornment on NeSI. I made a few changes for the issues I noticed while going through this page.

File: workflow/environments/README.md (hunk 1 above)
Comment (AndrewRidden-Harper) [reply workflow/environments/README.md]: I agree that changing this to include the full path might be better to avoid potential confusion

**Review by AndrewRidden-Harper — APPROVED** [review]
Looks good. My only comment is that I agree with Jake's comment that using the full path might be better to avoid potential confusion

**Pull request merged**